style.css

```
@@ -164,4 +164,87 @@ html {
 .social-icon {
   width: 20px;
   height: 20px;
-}
\ No newline at end of file
+}
+
+.feature-section-reverse {
+  display: flex;
+  margin: 138px auto;
+  gap: 64px;
+  justify-content: space-between;
+}
+
+.contents-reverse {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
```

❗️ 수정요청
index 페이지의 해당 요소가 디자인대로 보일 수 있도록 이미지와 동일한 width를 주시는 것이 좋을 것 같아요.
.contents와 .contents-reverse에게 아래 속성을 추가해주시는 것을 추천드려요.

```suggestion
.contents, .contents-reverse {
  display: flex;
  flex-direction: column;
  justify-content: center;
  max-width: 560px; // 최대 너비는 이미지와 동일하게 설정
  width: 100%;
}

```